builder/vmware/common/step_configure_vmx.go

```diff
@@ -110,7 +110,7 @@ func (s *StepConfigureVMX) Run(ctx context.Context, state multistep.StateBag) mu
 	} else {
 		displayName, ok := vmxData["displayname"]
 		if !ok { // Packer converts key names to lowercase!
-			err := fmt.Errorf("Error: Could not get value of displayName from VMX data")
+			err := fmt.Errorf("error returning value of displayName from vmx data")
```

Comment:
Capitalisation here?
```suggestion
			err := fmt.Errorf("error returning value of displayName from VMX data")
```

Reply:
Resolved in 67054cd.